examples/index.json

```
@@ -1413,6 +1413,14 @@
                 "verifyOutput": false,
                 "isLearnByExample": false
             },
+            {
+                "name": "Retry",
```

```suggestion
                "name": "Retry with Client",
```

just a suggestion to improve the consistency

---

Removed that `client` part because always retry will only gets applied to the client. And the HTTP also has named it `Retry`. WDYT?